frontend/http/frontend.go

```diff
@@ -144,7 +144,8 @@ func (t *Frontend) ListenAndServe() error {
 	}
 
 	// Start the HTTP server and gracefully handle any network errors.
-	if err := t.s.ListenAndServe(); err != nil {
+	if err := t.s.ListenAndServe(); err != nil && err != http.ErrServerClosed {
+		// TODO(mrd0ll4r): Do we still need this?
```

Comment:
I don't think so. Let's just log the error here and not bother checking the net.OpError stuff. That was all specific to graceful